app/models/user.rb

```diff
@@ -1,4 +1,8 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
- [ ] Great job so far team, however, it's required for the application to ask for confirmation by email. Please let's address this. 💯 👌🏻 
- You can check the `:confirmable` and enable it in the user model.

Reply:
I later realized this, i added the :confirmable and after signinup i logged in but i wasn't redirected, to home any tip please?